src/styles/index.css

```diff
@@ -0,0 +1,68 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: lightblue;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+h1 {
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.product-card {
+  width: 200px;
+  background-color: #fff;
```

Comment:
Consider using SCSS variables for colors like `background-color` and `border-color` to make it easier to manage and update these values.